### ISSUE TITLE
Mob Error

### ISSUE THREAD

**Issue by ghost** [mangosArchives/databaseZero_Rel19#46]
[The Forlorn Spirits](http://www.wowhead.com/npc=2044) that spawn throughout the Legend of Stalvan (66-98) questline spawn as neutral and do not automatically engage in combat.

**Comment by Ghurok** [comment]
According to wowpedia she should be neutral but still, oddly enough, attack you.

http://www.wowpedia.org/index.php?title=Forlorn_Spirit&oldid=1805958 (2009 though)

**Comment by cala** [comment]
In order to allow the creature to attack the player, you need the following query:

``` SQL
INSERT INTO `dbscripts_on_quest_start` (`id`, `delay`, `command`, `datalong`, `datalong2`, `buddy_entry`, `search_radius`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `comments`) VALUES
(NN, 3, 26, 0, 0, 2044, 25, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'forces Forlorn Spirit to attack player');
```

Where NN is the quest entry for the related quest (there are two quests where creature 2044 is spawned at start). As this is a spawned creature, this is the cleanest workaround I can think of for this special case of a neutral creature that must attack on sight.

**Comment by danielsreichenbach** [comment]
This requires a bit of research, and thus will be scheduled for release 2.0.8, since it can not be resolved for 2.0.7.

**Comment by Evildead666** [comment]
Fixed in commit https://github.com/mangoszero/database/commit/bbc13675243996064f10bfa04e45761edb5b0c6c and https://github.com/mangoszero/database/commit/3aa3b730674136a048ceec761cd6fa7e36c0765e
